mm2src/coins/utxo/utxo_withdraw.rs

```diff
@@ -213,6 +226,69 @@ where
     }
 }
 
+/// Checks if fee priorities are defined in the coin's configuration,
+/// retrieves the fee estimate from mempool, and adjusts the fee for Bitcoin if necessary.
+async fn generate_withdraw_fee_using_priority<C: UtxoCommonOps + GetUtxoListOps>(
+    coin: &C,
+    priority: &UtxoFeePriority,
+    priorities: &Option<UtxoFeePriorities>,
+    decimals: u8,
+) -> Result<u64, MmError<WithdrawError>> {
+    // Check if priorities are defined in coin config.
+    let Some(detail) = priorities else {
+         return MmError::err(WithdrawError::InternalError(format!("fee_priorities not found in {} config",  coin.as_ref().conf.ticker)))
+     };
+    let n_blocks = match priority {
+        UtxoFeePriority::Low => detail.low,
+        UtxoFeePriority::Normal => detail.normal,
+        UtxoFeePriority::High => detail.high,
+    };
+    let mut estimate_fee_sat = coin
+        .as_ref()
+        .rpc_client
+        .estimate_fee_sat(
+            decimals,
+            &crate::utxo::rpc_clients::EstimateFeeMethod::Standard,
+            &None,
+            n_blocks.into(),
+        )
+        .compat()
+        .await?;
+
+    if ["BTC"].contains(&coin.as_ref().conf.ticker.as_str()) {
```

Comment:
Is this intended to check tickers like that?
I guess this would be false for coins like "BTC-segwit"
(Maybe consider adding a config param?)